src/sections/product-main-template.liquid

```diff
@@ -350,7 +352,7 @@
       </li>
       {% endunless %}
       {% unless noreviews %}
-      <li>
+      <li id="reviewsTab">
```

Comment:
the linting is going to start enforcing our css id/class naming, I'm aware what we have a lot of camel case legacy IDs we'll need to address but anything new let's start using kebab.  https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters